### ISSUE TITLE
Case sensitivity issue for instructor Google ID

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#1135]
_From [OTS.Nexus](https://code.google.com/u/OTS.Nexus/) on August 15, 2013 23:53:21_

This special case is rarely to be happened but should be taken care when admin creates a new instructor:

The Google ID must be entered in the same capitalization as the original one in Google account, or else the instructor will not be able to be log in to the system. For example, my Google ID is OTS.Nexus but if I entered ots.nexus when creating an instructor account, the system will not allow me to log in.

In other words, the current system is case sensitive for Google ID.
(although Google itself are not case sensitive)

_Original issue: http://code.google.com/p/teammatespes/issues/detail?id=1136_
